examples/platforms/utils/mac_frame.cpp

```diff
@@ -306,9 +316,15 @@ otError otMacFrameProcessTransmitSecurity(otRadioFrame *aFrame, otRadioContext *
     otMacKeyMaterial *key = nullptr;
     uint8_t           keyId;
     uint32_t          frameCounter;
+    bool              processKeyId;
+
+    processKeyId =
+#if OPENTHREAD_CONFIG_WAKEUP_END_DEVICE_ENABLE
```

Comment:
Hi @zhanglongxia, shouldn't this be `OPENTHREAD_CONFIG_WAKEUP_COORDINATOR_ENABLE`? Since we are in the transmission path of the wake-up frame.

Reply:
@edmont Thanks for pointing out this issue. I have pushed a commit (https://github.com/openthread/openthread/pull/11047) to correct it.